liver_imaging_analysis/engine/engine.py

```diff
@@ -338,6 +338,7 @@ def fit(
         batch_callback_epochs = None,
         save_weight = False,
         save_path = config.save["potential_checkpoint"],
+        metric_epoch =1
```

Comment:
why are you adding another parameter for metric? can't you use evaluate_epochs for this purpose?